spread.yaml

```diff
@@ -146,25 +146,10 @@ prepare: |
     # utilities
     apt-get install -y devscripts expect gdebi-core jq rng-tools software-properties-common git
 
-    # needed so that we have golang-gopkg-macaroon.v1 which is not (yet)
-    # in trusty
-    add-apt-repository --update ppa:snappy-dev/image
-    # this should not be needed but apparently it is :/
-    apt-get update
-    # in 16.04: apt build-dep -y ./
```

Comment:
I think this gives us go and a few other things. The tests fail on (amongst others perhaps) lack of go